vendor/ddcloud/resource_additional_nics.go

```diff
@@ -0,0 +1,75 @@
+package ddcloud
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+const (
+	resourceKeyNicServerID     = "serverid"
+	resourceKeyNicID           = "id"
```

Comment:
You don't need to declare the `id` property on resources; it's built-in, and accessed via the `.GetID()` and `.SetID()` functions.